test/utils/windowUtilsTests.ts

```diff
@@ -0,0 +1,58 @@
+///<reference path="../testReference.ts" />
+
+var assert = chai.assert;
+
+describe("Utils.Window", () => {
+  it("deprecated()", () => {
+    var warningTriggered = false;
+    var oldWarn = Plottable.Utils.Window.warn;
+    Plottable.Utils.Window.warn = (msg: string) => {
+      warningTriggered = true;
+    };
+
+    Plottable.Utils.Window.deprecated("v0.77.2");
+
+    assert.isTrue(warningTriggered, "the warning has been triggered");
```

Comment:
Can we get better information from the `deprecated` call?  We want the warning to trigger, but we also want it to trigger with the methodName in mind right?